### DISCUSSION TITLE
Does Kurtosis support other container registries or libraries?

### DISCUSSION THREAD

**Discussion by guisellaDes** [kurtosis-tech/kurtosis#1273] [Q&A] [answered]
Does Kurtosis support other container registries or libraries?

**Comment by leeederek** [comment, marked as answer]
Hi @guisellaDes!

Currently, Kurtosis supports any **public** container registry (Dockerhub, Google Cloud Container Registry, ECR, etc.). At this time, Kurtosis does not support private container registries and so if you need to use a private container image, we recommend you build it locally and Kurtosis will automatically use the locally built image when it cannot find the tag in a remote public registry. 

If your project or team requires using a private container registry, please let us know by [filing an issue in our Github](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&projects=&template=feature-request.yml) or letting us know in [Discord](https://discord.gg/jJFG7XBqcY).

Thank you!
